frontend/app/routes/_gcweb-app.personal-information.home-address.address-accuracy.tsx

```diff
@@ -0,0 +1,74 @@
+import { type ActionFunctionArgs, type LoaderFunctionArgs, json, redirect } from '@remix-run/node';
+import { Form, Link, useLoaderData } from '@remix-run/react';
+
+import clsx from 'clsx';
+import { useTranslation } from 'react-i18next';
+
+import { Address } from '~/components/address';
+import { getSessionService } from '~/services/session-service.server';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('personal-information');
+
+export const handle = {
+  breadcrumbs: [
+    { labelI18nKey: 'personal-information:home-address.address-accuracy.breadcrumbs.home', to: '/' },
+    { labelI18nKey: 'personal-information:home-address.address-accuracy.breadcrumbs.personal-information', to: '/personal-information' },
+    { labelI18nKey: 'personal-information:home-address.address-accuracy.page-title' },
+  ],
+  i18nNamespaces,
+  pageIdentifier: 'CDCP-0013',
+  pageTitleI18nKey: 'personal-information:home-address.address-accuracy.page-title',
+};
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  const sessionService = await getSessionService();
+  const session = await sessionService.getSession(request.headers.get('Cookie'));
+  const newHomeAddress = await session.get('newHomeAddress');
+  return json({ newHomeAddress });
+}
+
+export async function action({ request }: ActionFunctionArgs) {
+  return redirect('/personal-information/home-address/confirm');
+}
+
+export default function PersonalInformationHomeAddressAccuracy() {
+  const { newHomeAddress } = useLoaderData<typeof loader>();
+  const { t } = useTranslation(i18nNamespaces);
+  return (
+    <>
+      <Form method="post">
+        <section className="alert alert-warning mt-4">
+          <h3>{t('personal-information:home-address.address-accuracy.invalid-address')}</h3>
```

Comment:
We currently go from an `<h1>` (the page title) to an `<h3>`.

```suggestion
          <h2>{t('personal-information:home-address.address-accuracy.invalid-address')}</h3>
```

This same comment applies for the mailing address route.

Reply:
Changed to <h2>